dinh_ngoc_do/View_DB.php

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" 
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+    <link href="style_ViewDB.css">
+    <title>View_DB</title>
+</head>
+<body>
+    <h1 style="text-align: center;">List User</h1>
+    <div class="container">
+        <div class="wrapper">
+            <a href="Action_ViewDB.php" name="deletealluser" class="btn btn-danger">Delete All</a>
```

Comment:
ở đây nếu e dùng GET thì phải truyền tham số `deletealluser` chứ thẻ a làm gì có attribute `name`
```suggestion
            <a href="Action_ViewDB.php?deletealluser=1" name="deletealluser" class="btn btn-danger">Delete All</a>
```